core/src/main/scala/io/scalac/mesmer/core/module/AkkaDispatcherModule.scala

```diff
@@ -0,0 +1,79 @@
+package io.scalac.mesmer.core.module
+
+import com.typesafe.config.{Config => TypesafeConfig}
+import io.scalac.mesmer.core.model.Version
+import io.scalac.mesmer.core.typeclasses.{Combine, Traverse}
+import io.scalac.mesmer.core.util.LibraryInfo.LibraryInfo
+
+trait AkkaDispatcherConfigMetricsModule extends MetricsModule {
+  this: Module =>
+
+  override type Metrics[T] <: AkkaDispatcherConfigMetricsDef[T]
+
+  trait AkkaDispatcherConfigMetricsDef[T] {
+    def minThreads: T
+    def maxThreads: T
+    def parallelismFactor: T
+  }
+
+}
+
+object AkkaDispatcherModule extends MesmerModule with RegistersGlobalConfiguration with AkkaDispatcherConfigMetricsModule {
+
+  final case class Impl[T](minThreads: T, maxThreads: T, parallelismFactor: T)
+    extends AkkaDispatcherConfigMetricsDef[T]
+
+  final case class AkkaDispatcherJars[T](akkaActor: T, akkaActorTyped: T) extends Module.CommonJars[T]
+
+  override type Metrics[T] = AkkaDispatcherConfigMetricsDef[T]
+  override type All[T]     = Metrics[T]
+  override type Jars[T]    = AkkaDispatcherJars[T]
+
+  override def defaultConfig: Config = Impl[Boolean](true, true, true)
+
+  override protected def extractFromConfig(config: TypesafeConfig): Config = {
```

Comment:
which config is getting passed here? Is it the akka config from application.conf ?